runtime/port/sysvipc/j9shmem.c

```diff
@@ -1348,23 +1348,8 @@ j9shmem_createDir(struct J9PortLibrary* portLibrary, char* cacheDirName, uintptr
 		Trc_PRT_j9shmem_createDir_ExitFailedFile();
 		break;
 	case EsIsDir:
-#if !defined(J9ZOS390)
-		if (J9SH_DIRPERM_ABSENT == cacheDirPerm) {
-			/* cacheDirPerm option is not specified. Change permission to J9SH_DIRPERM. */
-			rc2 = changeDirectoryPermission(portLibrary, cacheDirName, J9SH_DIRPERM);
```

Comment:
I think this behavior is still valid, but perhaps only when using a default /tmp/javasharedresources directory. I don't think it should be removed completely. Otherwise it could stop other users on the machine from creating or using caches.